mw-webapp/src/component/footer/Footer.module.scss

```diff
@@ -1,36 +1,44 @@
 @import "src/styles/variables";
 
-$userNameFontSize: 16px;
+$logoHeight: 55px;
+$privacyPaddingTop: 10px;
```

Comment:
Let's move padding = 10px to global var and to tokens in figma.

Reply:
Waiting for a response from the designer.

Reply:
Designer decided to apply 8px for padding top and bottom